### PR TITLE
[2.11.x] DDF-3374 Use a separate build option for webpack dev and prod mode

### PR DESCRIPTION
**Pull request by lessarderic**
#### What does this PR do?
Adds new `webBuildEnv` option for webpack that is independent of the `skipStatic` option.
Also upgraded `gitflow-incremental-builder` plugin to version 3.3.

#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@djblue 
@oconnormi  

#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@andrewkfiedler
@rzwiefel

#### How should this be tested? (List steps with links to updated documentation)
* Build with `-DwebBuildDev=dev` and make sure webpack uses the development configuration.
* Build with `-DwebBuildDev=prod` and make sure webpack uses the production configuration.
* Build without the `-DwebBuildDev` option and make sure webpack uses the production configuration.

#### What are the relevant tickets?
[DDF-3374](https://codice.atlassian.net/browse/DDF-3374)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -227,15 +228,13 @@
...
         <spark.version>2.5.5</spark.version>
         <sardine.version>5.7</sardine.version>
+        <xerces.version>2.11.0</xerces.version>
 
         <!-- gitflow-incremental-builder -->
         <!-- See https://github.com/vackosar/gitflow-incremental-builder for a list of all the options -->
         <gib.referenceBranch>refs/remotes/origin/master</gib.referenceBranch>
         <gib.baseBranch>HEAD</gib.baseBranch>
         <gib.enabled>false</gib.enabled>
-        <commons-lang3.version>3.4</commons-lang3.version>
```
Comment (rzwiefel) [inline pom.xml]: Don't need commons-lang3 property anymore?

Comment (lessarderic) [reply pom.xml]: It was in there twice :-)

**Comment by lessarderic** [comment]
build now

**Comment by lessarderic** [comment]
build now

**Pull request merged**